client/components/forms/form-text-input-with-action/index.jsx

```diff
@@ -0,0 +1,129 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+import { keys, omit, noop } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import FormTextInput from 'components/forms/form-text-input';
+import FormButton from 'components/forms/form-button';
+
+export default React.createClass( {
```

Comment:
I thought Facebook was going to wait until decorators were stabilized. For the ES6 classes, what are we using for the bound methods?

Didn't know about the `displayName` transform, though, removing that.